Rs/intro (#121)
neuralmagic/docs: src/content/products/deepsparse/community.mdx

ONNX gives the flexibility to serve your model in a framework-agnostic environment.
Support includes [PyTorch,](https://pytorch.org/docs/stable/onnx.html) [TensorFlow,](https://github.com/onnx/tensorflow-onnx) [Keras,](https://github.com/onnx/keras-onnx) and [many other frameworks](https://github.com/onnx/onnxmltools).

DeepSparse is available in two editions:
1. [**DeepSparse Community**](#installation) is open-source and free for evaluation, research, and non-production use with our [Engine Community License](https://neuralmagic.com/legal/engine-license-agreement/).
2. [**DeepSparse Enterprise**](https://docs.neuralmagic.com/products/deepsparse-ent) requires a Trial License or [can be fully licensed](https://neuralmagic.com/legal/master-software-license-and-service-agreement/) for production, commercial applications.

## Features

- 🔌 [DeepSparse Server](https://github.com/neuralmagic/deepsparse/tree/main/src/deepsparse/server)
- 📜 [DeepSparse Benchmark](https://github.com/neuralmagic/deepsparse/tree/main/src/deepsparse/benchmark)
- 👩‍💻 [NLP and Computer Vision Tasks Supported](https://github.com/neuralmagic/deepsparse/tree/main/examples)

> ## 🧰 Hardware Support and System Requirements

Reviewer: ```suggestion
## Hardware Support and System Requirements
```

Review [CPU Hardware Support for Various Architectures](https://docs.neuralmagic.com/deepsparse/source/hardware.html) to understand system requirements.
DeepSparse works natively on Linux. Mac and Windows require running Linux in a Docker or virtual machine; it will not run natively on those operating systems.

> DeepSparse is tested on Python 3.7-3.10, ONNX 1.5.0-1.12.0, and ONNX opset version 11+, and manylinux compliant.

Reviewer: ```suggestion
DeepSparse is tested on Python 3.7-3.10, ONNX 1.5.0-1.12.0, and ONNX opset version 11+. It is manylinux compliant.
```

Reviewer: The reason Beth made this edit is because you wouldn't say "tested on manylinux compliant" which is how the sentence read. If we were to keep it as is, it would need to read 
", and manylinux compliant systems." or what she offered for example.

## 🧰 Hardware Support and System Requirements

Review [CPU Hardware Support for Various Architectures](https://docs.neuralmagic.com/deepsparse/source/hardware.html) to understand system requirements.
DeepSparse works natively on Linux; Mac and Windows require running Linux in a Docker or virtual machine; it will not run natively on those operating systems.

DeepSparse is tested on Python 3.7-3.10, ONNX 1.5.0-1.12.0, ONNX opset version 11+, and manylinux compliant.
Using a [virtual environment](https://docs.python.org/3/library/venv.html) is highly recommended.

## Installation

Install DeepSparse Community as follows:

```bash
pip install deepsparse
```

See the [DeepSparse Community Installation Page](https://docs.neuralmagic.com/get-started/install/deepsparse) for further installation options.

To trial or inquire about licensing for DeepSparse Enterprise, see the [DeepSparse Enterprise documentation](https://docs.neuralmagic.com/products/deepsparse-enterprise).

> ## Features

Reviewer: ```suggestion
## Components
```

Reviewer: Per Brian request/alignment

### Scheduling Single-Stream, Multi-Stream, and Elastic Inference

> DeepSparse offers up to three types of inferences based on your use case. Read more details here: [Inference Types](https://github.com/neuralmagic/deepsparse/blob/main/docs/source/scheduler.md).

Reviewer: ```suggestion
DeepSparse offers up to three types of inferences based on your use case. Read [more about inference types](https://github.com/neuralmagic/deepsparse/blob/main/docs/source/scheduler.md).
```

Reviewer: Stylistically, avoid using the word filler "here" and tell the user where/what you want them to do really do.

Contribute with code, examples, integrations, and documentation as well as bug reports and feature requests! [Learn how here.](https://github.com/neuralmagic/deepsparse/blob/main/CONTRIBUTING.md)

For user help or questions about DeepSparse, sign up or log into our [Neural Magic Community Slack](https://join.slack.com/t/discuss-neuralmagic/shared_invite/zt-q1a1cnvo-YBoICSIw3L1dmQpjBeDurQ). We are growing the community member by member and happy to see you there. Bugs, feature requests, or additional questions can also be posted to our [GitHub Issue Queue.](https://github.com/neuralmagic/deepsparse/issues) You can get the latest news, webinar and event invites, research papers, and other ML performance tidbits by [subscribing](https://neuralmagic.com/subscribe/) to the Neural Magic community.

For more general questions about Neural Magic, complete this [form.](http://neuralmagic.com/contact/)

### License

> The Community Edition of the project's binary containing the DeepSparse Engine is licensed under the [Neural Magic Engine License.](https://github.com/neuralmagic/deepsparse/blob/main/LICENSE-NEURALMAGIC)

Reviewer: ```suggestion
DeepSparse Community is licensed under the [DeepSparse Community License.](https://github.com/neuralmagic/deepsparse/blob/main/LICENSE-NEURALMAGIC)
```

> Example files and scripts included in this repository are licensed under the [Apache License Version 2.0](https://github.com/neuralmagic/deepsparse/blob/main/LICENSE) as noted.

Reviewer: ```suggestion
Some source code, example files, and scripts included in the DeepSparse GitHub repository or directory are licensed under the [Apache License Version 2.0](https://github.com/neuralmagic/deepsparse/blob/main/LICENSE) as noted.
```
